tests/integration/test_storage_azure_blob_storage/test.py

```diff
@@ -69,19 +71,27 @@ def azure_query(
             continue
 
 
-def get_azure_file_content(filename):
+def get_azure_file_content(filename, port):
     container_name = "cont"
-    connection_string = "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;"
+    connection_string = (
```

Comment:
Is it the same as above? What about moving it to Cluster?

Reply:
ya good idea. Moved connection string & account url to cluster